atomic_reactor/plugins/exit_koji_import.py

```diff
@@ -123,21 +127,107 @@ def __init__(self, tasker, workflow, kojihub=None, url=None,
         self.osbs = get_openshift_session(self.workflow, self.openshift_fallback)
         self.build_id = None
         self.session = None
+        self.source_build = bool(self.workflow.build_result.oci_image_path)
```

Comment:
well if source build plugin finishes and has results we are sure we are doing source container build, I am not aware of any other better thing to check

but since this depends on output of build plugin, I'd say it is safe

Reply:
I'm thinking about adding `kind` to attribute to workflow and methods `is_image_build`, `is_source_container_build` WDYT?

Reply:
I mean sure, but still buildResults are quite conclusive